internal/payloads/launch_template_payload.go

```diff
@@ -14,28 +15,21 @@ type LaunchTemplateResponse struct {
 }
 
 type LaunchTemplateListResponse struct {
-	Data []*LaunchTemplateResponse `json:"data" yaml:"data"`
-}
-
-func (s *LaunchTemplateResponse) Bind(_ *http.Request) error {
-	return nil
-}
-
-func (s *LaunchTemplateResponse) Render(_ http.ResponseWriter, _ *http.Request) error {
-	return nil
+	Data     []*LaunchTemplateResponse `json:"data" yaml:"data"`
+	Metadata page.Metadata             `json:"metadata" yaml:"metadata"`
```

Comment:
Ah here it is, so the root is correct actually. Tests need to be updated I guess.